tests/test_correct_entries.py

```diff
@@ -0,0 +1,16 @@
+import argparse
+from pathlib import Path
+
+import pytest
+
+from validator.checker import UpdateChecker
+from validator.utils import get_latest_version
+
+@pytest.mark.parametrize("version", range(1,get_latest_version()+1))
```

Comment:
nitpick: style
```suggestion
@pytest.mark.parametrize('version', range(1, get_latest_version() + 1))
```

flake8 would have hit here. We should probably activate it here too